src/controllers/transaction.controllers.js

```diff
@@ -21,12 +21,26 @@ const addTransaction = async (buyerId, sellerId, houseId, price) => {
 export const buyHouse = async (data) => {
   const { buyerId, houseId, price } = data;
```

Comment:
`buyerId` and `houseId` don't exist in `data`. Something like this should solve the problem: `const { buyer: buyerId, house: houseId, price } = data;`.

Reply:
It's only used in POST _transactions/_  route and the data parameter is actually req.body.

Reply:
I see. These are the keys of data:
![Screenshot from 2022-03-06 13-52-33](https://user-images.githubusercontent.com/47178494/156924078-7eb25e55-9058-4b43-bcef-c171866c7d8f.png)
Destructed `buyerId` and `houseId` aren't present in the data so they are `undefined`. What I suggest is to destructure `buyer` and `house` from the data and alias them to `buyerId`, `houseId` respectively.

Reply:
<img width="554" alt="image" src="https://user-images.githubusercontent.com/80357638/156924512-1838d219-79d2-43c5-87ff-5dcf10bc0d77.png">

This data contains only strings with ids of documents necessary to create transaction. It's not representing the data in the database.

Reply:
I see. My bad. I thought you defined the keys but it was me. I'm sorry for the hassle.